cmd/jet/main.go

```diff
@@ -71,6 +75,10 @@ func init() {
 	flag.StringVar(&ignoreEnums, "ignore-enums", "", `Comma-separated list of enums to ignore`)
 
 	flag.StringVar(&destDir, "path", "", "Destination dir for files generated.")
+	flag.StringVar(&modelPkg, "model-pkg", "", "Package name for the model files generated")
```

Comment:
We should be able to pass a relative path not just package name. 
Maybe usage should say something like:  `Model files package relative path to destination directory`.
Also default value is missing.

Reply:
Initially, I was only thinking about allowing the customization of the package name.
But yeah, this would make it more flexible.

Will do.

Reply:
Descriptions were incorrect as well, fixed.

Reply:
Changing the description of `destDir` to replace `Destination dir` with `Destination directory` for consistency with the new flags.

Reply:
Adding a period at the end of the usage description of `-ignore-tables`, `-ignore-views`, and `-ignore-enums` flags to make them consistent with all the other flags.